src/PathItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OpenApiPropTypes from './open-api-prop-types';
```

Comment:
@brendanthomas1 after seeing the indirection here and working through some of the other examples I don't think reuse is as big of a deal. If we took that idea to the logical conclusion I think we would end up with one giant OPenAPI prop type shape which seems kind of silly 😆 .

What do you think about just moving them to the top of the component files?